src/NUnitConsole/nunit3-console/ResultSummary.cs

```diff
@@ -74,6 +74,22 @@ public int NotRunCount
         }
 
         /// <summary>
+        /// Returns the number of failed test cases (including errors and invalid tests)
+        /// </summary>
+        public int FailedCount
+        {
+            get { return FailureCount + InvalidCount + ErrorCount;  }
```

Comment:
At some (maybe future) point, I think we want to just add up the test cases with a status of failed. That way, when we add new kinds of failed tests, the total will continue to be correct. Same for the next one.